On Critical Learning Theory: the hidden curriculum (#4754)
galaxyproject/training-material: topics/contributing/tutorials/critical_theory/tutorial.md

---
layout: tutorial_hands_on

title: "On Critical Learning Theory: the Hidden Curriculum"
level: Advanced
subtopic: pedagogy
tags:
    - critical learning theory
    - pedagogy
    - philosophy
time_estimation: 2h
questions:
- What is Critical Learning Theory about?
- What is its relation with the Critical Theory developed by the Frankfurt School?
- How is possible to apply the principles of Critical Learning Theory?
objectives:
- To introduce the historical origins of Critical Learning Theory.
- Systematically present the main methods associated with Critical Learning Theory.
contributions:
  authorship:
    - gallardoalba
---

> <warning-title>Recommended starting point</warning-title>

> > This training can be considered as the continuation of [Principles of learning and how they apply to training and teaching ]({% link topics/contributing/tutorials/learning-principles/tutorial.md %}). If you wish to have a panoramic view of the theoretical corpus in which the **Critical Learning Theory** is embedded, we recommend to start with the above-mentioned training.

Reviewer: please add a `follow_up_training` to that tutorial and a requirements to this one like

```
requirements:
  - type: "internal"
    topic_name: contributing
    tutorials:
      - learning-principles
```

though this (and the mentioned tutorial) both feel more like they belong in the `teaching` topic (@bebatut what do you think)

{: .warning}

# Introduction

The **Critical Learning Theory**  is a pedagogical paradigm rooted in the broader **Critical Theory** tradition originated with the **Frankfurt School** of thought, formally known as the Institute for Social Research (*Institut für Sozialforschung*), an academic hub founded in 1923 at Goethe University in Frankfurt, Germany. 

Consisting of a renowned collective of critical theorists such as Max Horkheimer, Theodor Adorno, Walter Benjamin, and Herbert Marcuse, the **Frankfurt School** developed the **Critical Theory** as a multidisciplinary approach crafted with the intent to comprehend and actively confront the oppressive dynamics ingrained within industrial societies (fig. 1).

> ![Frankfurt School picture](../../images/critical_theory/frankfurt_school.png "Left image: Max Horkheimer and Theodor Adorno in Heidelberg in 1964 at the Max Weber Institute of Sociology. Right image: Herbert Marcuse in Newton, Massachusetts in 1955. Source: Wikipedia")

Reviewer: ```suggestion
![Frankfurt School picture, on the left two old bald white men dressed in suits shake hands, right an old white man relaxes in a lawnchair holding a cigar.](images/critical_theory/frankfurt_school.png "Left image: Max Horkheimer and Theodor Adorno in Heidelberg in 1964 at the Max Weber Institute of Sociology. Right image: Herbert Marcuse in Newton, Massachusetts in 1955. Source: Wikipedia")
```

> *The findings are striking: almost 40 percent of global employment is exposed to AI. Historically, automation and information technology have tended to affect routine tasks, but one of the things that sets AI apart is its ability to impact high-skilled jobs.*
>
> — Kristalina Georgieva, managing director of the International Monetary Fund.
>
{: .quote}

## Educative systems

According the **Critical Theory**, educative systems are agents of ideological control linked to the principles and processes that govern the workplace; the dominant culture in educative systems is characterised by selectively ordering and legitimising privileged forms of language, social relations, life experiences and modes of reasoning ({% cite Marcuse1964 %}).

> The philospher Pedro García Olivo affirms that the criterion governing Western educative systems, in line with OECD recommendations, is none other than that of favouring the adaptation of students to the requirements of the established productive and political apparatus, which requires their psychological and cultural homogenisation ({% cite Olivo2005 %}).

Reviewer: perhaps write `{OECD}` here and add 

```
abbreviations:
  OECD: Organisation for Economic Co-operation and Development
```

to the yaml header.

In the face of the conservative claim that educative systems transmit objective knowledge, **Critical Learning Theory** concives knowledge as a particular representation of the dominant culture that structure the hegemonic cosmovision throught a selective process of emphasis and exclusions ({% cite Giroux1988 %}).

# Critical thinking

> *Questioning is an exercise in dismantling those certainties that are installed as layers of truths imposing the tyranny of the obvious.*
>
> — Dario Sztajnzrajbe, ¿Para qué sirve la filosofía?
{: .quote}

Before going deeper, it is inportant to accurately define **critical thinking**, since it is usually concived as a nebulose concept whose applications appears to be limitated to specific situations.

> The most common definition comes from the **positivist tradition** in the applied sciences; it appeals to the function of teaching students to analyse and develop reading and writing tasks from the perspective of formal logical patterns of coherence, and determining if the conclusions can be derivated from the evaluated data ({% cite Giroux1988 %}). Is it enought?

Reviewer: ```suggestion
The most common definition comes from the **positivist tradition** in the applied sciences; it appeals to the function of teaching students to analyse and develop reading and writing tasks from the perspective of formal logical patterns of coherence, and determining if the conclusions can be derivated from the evaluated data ({% cite Giroux1988 %}). Is it enough?
```

> <comment-title>Frankfurt School and critical thinking </comment-title>
>
> Some critical theorists of the first generation, including Max Horkheimer, Theodor Adorno and Walter Benjamin, argued that in Western democracies the capacity for critical reasoning is rapidly being eclipsed.
>
> Pointing to developments such as the intrusion of the state, the culture industry and the concentration of wealth in fewer and fewer people, these thinkers fear that the ideological and material conditions that make public interaction and critical thinking possible are being undermined by the increasing standardisation, fragmentation and commercialisation of everyday life.
>
{: .comment}

From the assumption of this conception of critical thinking a new question arises: is it really in the interest of education system to encourage critical thinking? One element that can help to answer this question is the hidden curriculum.

> <question-title>Critical thinking quiz</question-title>
>

> > 1. **How does critical thinking align with the principles of both Critical Theory and Critical Learning Theory?** <br>
> > &nbsp; a. By avoiding critical examination of societal issues. <br>
> > &nbsp; b. By promoting conformity to existing structures. <br>
> > &nbsp; c. By fostering a reflective approach to learning and encouraging critical analysis. <br>
> > &nbsp; d. By discouraging diverse perspectives.

Reviewer: ```suggestion
> 1. **How does critical thinking align with the principles of both Critical Theory and Critical Learning Theory?**
>    a. By avoiding critical examination of societal issues.
>    b. By promoting conformity to existing structures.
>    c. By fostering a reflective approach to learning and encouraging critical analysis.
>    d. By discouraging diverse perspectives.
```

please use proper markdown formatting here. We can style it as a/b/c/d with CSS which feels a bit safer.

For this page we could add a bit like

```
<style>
blockquote.question ol ol li {
  list-style: lower-alpha;
}
</style>
```

in order to achieve that.

Also it's a *lot* of bold, maybe conider not bolding the questions?

## Technological system

The technological system is a complex arrangement of interconnected components, including hardware, software, data, and human operators, that work together to perform a specific function or achieve a particular goal. According the philospher Günther Anders, **the design of technological systems goes beyond mere neutrality in information transmission**; instead, they actively convey values and thought patterns ({% cite Anders2002 %}, {% cite Gallardo2021 %}).

> <comment-title>Notes on Günther Anders</comment-title>
>
> Günther Anders (12 July 1902 - 17 December 1992), a German-born philosopher, journalist and critical theorist, stands as a prominent figure known for his insightful reflections on technology, existentialism, and the human condition. Although Anders did not become a formal member of the **Frankfurt School**, he is considered to have exerted an important influence on some of its members, with whom he maintained a close relationship throughout his life.
>
> In 1923, Gunther Anders obtained his Ph.D. in philosophy from the University of Freiburg, having studied under the esteemed philosophers Martin Heidegger and Edmund Husserl.
>

> > ![Anders picture](../../images/critical_theory/anders.png "Günther Anders. Source: soziologie.uni-freiburg.de")

Reviewer: ```suggestion
> ![An old white man wearing glasses and a suit, stares off to the right side with a very pensive thoughtful expression.](../../images/critical_theory/anders.png "Günther Anders. Source: soziologie.uni-freiburg.de")
```

> <comment-title>Biology and ideology</comment-title>
>
> According to Richard Lewontin and Richard Levins, theories of evolution underwent a shift in the course of the 20th century from an emphasis on directionality to a focus on the state of equilibrium, which can only be understood as ideological in origin.
>
> > *Just as the ideas of change and movement were the intellectual foundations of the bourgeois revolution, for the legitimisation of the collapse of the old class relations, the consolidation of that revolution in the latter part of the 19th century and in the 20th century has required a different view of the world, in line with the stabilisation of society. The idea of change had to be repressed in science as it was in society. The result has been the emphasis in modern evolutionary theories on dynamic equilibrium.*
> >
> {: .quote author="Richard Lewontin and Richard Levins" cite="The dialectical biologist"}
>

> > Similarly, these authors consider that **the description of evolution as producing order is also deeply ideological**.  An alternative conception is represented by the *[disorder-function paradigm]({% link topics/transcriptomics/tutorials/differential-isoform-expression/slides.html#19})*. In this sense, evolution is neither a fact nor a theory, but a way of organising knowledge about the world.

Reviewer: ```suggestion
> Similarly, these authors consider that **the description of evolution as producing order is also deeply ideological**.  An alternative conception is represented by the *[disorder-function paradigm]({% link topics/transcriptomics/tutorials/differential-isoform-expression/slides.html %}#19)*. In this sense, evolution is neither a fact nor a theory, but a way of organising knowledge about the world.
```
